noxfiles/setup_tests_nox.py

```diff
@@ -1,5 +1,7 @@
 from typing import Optional
 
+from nox import Session
```

Comment:
an isort/formatting error that had been flying under the radar, exposed now because we're properly checking in CI